lib/ssg/downloader.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require 'json'
+require 'net/http'
+
+module Ssg
+  # Downloads SCAP datastreams from the SCAP Security Guide
+  # https://github.com/ComplianceAsCode/content
+  class Downloader
+    LATEST_RELEASE_URI = URI('https://api.github.com/repos'\
+                             '/ComplianceAsCode/content/releases/latest')
```

Comment:
Currently you can only download the latest release of the SSG. I don't know why someone would want an older release, but it's possible we may need to allow that in the future.

Reply:
Now you can use this rake task to download any version of any SSGs:

```sh
rake ssg:sync DATASTREAMS=v0.1.28:rhel6,latest:fedora,latest:rhel8
```